nodejs/webrtc-hello-again/server.js

```diff
@@ -0,0 +1,257 @@
+const fs = require('fs')
+const express = require('express')
+const BandwidthWebRTC = require('@bandwidth/webrtc')
+const BandwidthVoice  = require('@bandwidth/voice')
+const uuid = require('uuid');
+const dotenv = require('dotenv').config()
+const jwt_decode = require('jwt-decode')
+const app = express();
+const bodyParser = require('body-parser')
+app.use(bodyParser.json());
+
+
+// config
+const port = 3000
+const localDir = __dirname
+const accountId = process.env.ACCOUNT_ID
+
+// Global variables
+BandwidthWebRTC.Configuration.basicAuthUserName = process.env.USERNAME;
+BandwidthWebRTC.Configuration.basicAuthPassword = process.env.PASSWORD;
+var webRTCController = BandwidthWebRTC.APIController;
+
+BandwidthVoice.Configuration.basicAuthUserName = process.env.USERNAME;
+BandwidthVoice.Configuration.basicAuthPassword = process.env.PASSWORD;
+var voiceController = BandwidthVoice.APIController;
+
+// create a map of PSTN calls that will persist
+let calls = new Map()
+
+// track out session ID
+let sessionId = false;
+let callId = false;
+
+
+/**
+ * Serve the single page app
+ */
+app.get('/', (req, res) => {
+  // return the base page
+  res.sendFile(localDir +'/simple_page.html')
+});
+
+/**
+ * Setup the call and pass info to the browser so they can join
+ */
+app.get('/startBrowserCall', (req, res) => {
+  // create the session
+  var sessionBody = new BandwidthWebRTC.Session({ "tag" : "session-test"});
+
+  webRTCController.createSession(accountId, sessionBody, function(error, sessionResponse, context) {
+    // console.log("session info: ", sessionResponse, "\n")
+    saveSessionId(sessionResponse.id)
+
+    // create a participant for this browser user
+    var participantBody = new BandwidthWebRTC.Participant(
+      {
+        "tag" : uuid.v1(),
+        "publishPermissions" : ["AUDIO"]
+      });
+
+    webRTCController.createParticipant(accountId, participantBody, 
+      function(error, createResponse, context) {
+        // console.log("participant response info: ", createResponse)
+        var body = new BandwidthWebRTC.Subscriptions({"sessionId" : getSessionId()});
+        webRTCController.addParticipantToSession(accountId, getSessionId(), createResponse.participant.id, body, 
+          function(error, response, context) {
+            // now that we have added them to the session, we can send back the token they need to join
+            console.log("sending token to web participant")
+            res.send('{"token":"'+ createResponse.token +'"}')
```

Comment:
I would send back actual JSON here, rather than stringified JSON. It should make life easier for the browser code requesting this too.
```javascript
res.send({token: createResponse.token});
```